### ISSUE TITLE
Run issues after last merge (puzzle init)

### ISSUE THREAD

**Issue by vasiliosz** [robinandeer/puzzle#76]
After #70 merge, base `puzzle` cmd is not working for me. Commit 483557d9cb506cd72ff7eba1605c7d479b2b3c03 works. 

1) Missing requirement for sqlalchemy 

```
[VZMBP puzzle]$ puzzle
Traceback (most recent call last):
  File "/usr/local/bin/puzzle", line 9, in <module>
    load_entry_point('puzzle==0.0.1', 'console_scripts', 'puzzle')()
  File "/usr/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 558, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2682, in load_entry_point
    return ep.load()
  File "/usr/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2355, in load
    return self.resolve()
  File "/usr/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2361, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/Users/vaszac/proj/puzzle/puzzle/__main__.py", line 14, in <module>
    from .cli import cli
  File "/Users/vaszac/proj/puzzle/puzzle/cli.py", line 9, in <module>
    from .factory import create_app
  File "/Users/vaszac/proj/puzzle/puzzle/factory.py", line 5, in <module>
    from .settings import BaseConfig
  File "/Users/vaszac/proj/puzzle/puzzle/settings.py", line 5, in <module>
    from .plugins import VcfPlugin
  File "/Users/vaszac/proj/puzzle/puzzle/plugins/__init__.py", line 8, in <module>
    from .sql import Store as SqlStore
  File "/Users/vaszac/proj/puzzle/puzzle/plugins/sql/__init__.py", line 2, in <module>
    from .store import Store
  File "/Users/vaszac/proj/puzzle/puzzle/plugins/sql/store.py", line 9, in <module>
    from sqlalchemy import create_engine
ImportError: No module named sqlalchemy
```

2) After installing sqlalchemy separately, new error (on new clone now): Can't find gemini plugin
```
[VZMBP puzzle-dev]$ puzzle
Traceback (most recent call last):
  File "/usr/local/bin/puzzle", line 9, in <module>
    load_entry_point('puzzle==0.0.1', 'console_scripts', 'puzzle')()
  File "/usr/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 558, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2682, in load_entry_point
    return ep.load()
  File "/usr/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2355, in load
    return self.resolve()
  File "/usr/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2361, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/Users/vaszac/proj/puzzle-dev/puzzle/__main__.py", line 14, in <module>
    from .cli import cli
  File "/Users/vaszac/proj/puzzle-dev/puzzle/cli.py", line 9, in <module>
    from .factory import create_app
  File "/Users/vaszac/proj/puzzle-dev/puzzle/factory.py", line 5, in <module>
    from .settings import BaseConfig
  File "/Users/vaszac/proj/puzzle-dev/puzzle/settings.py", line 5, in <module>
    from .plugins import VcfPlugin
  File "/Users/vaszac/proj/puzzle-dev/puzzle/plugins/__init__.py", line 8, in <module>
    from .sql import Store as SqlStore
  File "/Users/vaszac/proj/puzzle-dev/puzzle/plugins/sql/__init__.py", line 2, in <module>
    from .store import Store
  File "/Users/vaszac/proj/puzzle-dev/puzzle/plugins/sql/store.py", line 17, in <module>
    from puzzle.plugins import VcfPlugin, GeminiPlugin, Plugin
ImportError: cannot import name GeminiPlugin
```

**Comment by moonso** [comment]
Thank you Vasilios

**Comment by moonso** [comment]
Fixed in #77
